continuous-integration/python3.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# This script runs a passing subset of tests under Python 3.
+
+python setup.py install
```

Comment:
Python is installed using conda, the python version is set in a env var in the travis matrix.